src/Tools/dotnet-dump/ExtensionCommands/ClrMDHelper.cs

```diff
@@ -0,0 +1,420 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Diagnostics.Runtime;
+using System;
+using System.Collections.Generic;
+using Microsoft.Diagnostics.DebugServices;
+
+namespace Microsoft.Diagnostic.Tools.Dump.ExtensionCommands
```

Comment:
I wonder if we can't come up with a better name for this namespace and the subdirectory they are in.  We already have "Commands" and to me the "Extension" part of "ExtensionCommands" does add enough to distinguish it.  Maybe "ExtendedCommands" or just add them to "Commands".

Reply:
I picked this name because the long term goal is to move them to an "extension assembly" once the extensibility mechanism is ready.

Reply:
Yes that makes sense.  The name is ok.